Gemfile.lock

```diff
@@ -46,6 +46,7 @@ PATH
   specs:
     appeals_api (0.0.1)
       rails (~> 5.2.3)
+      sidekiq
```

Comment:
Interesting, why the addition to the module if you don't mind me asking?

Reply:
It should have been added when he module started using sidekiq but was overlooked. I tried to get the setup to match vbs_documents

Reply:
Good to know, is this a situation where we should include all of the gems that the module depends on, not just sidekiq?

Reply:
I think we need a tech debt ticket to sweep all the modules for that actually. I only did sidekiq because of me working with it, however there are more gems we should list out but it would be some spelunking. I think all the modules could use that.